docs/source/DeveloperDocumentation/VUDRM-token.md

```diff
@@ -75,24 +186,24 @@ The `polbegin` and `polend` settings use the timezone set at the account level.
 
 There are also limitations depending on environments that are not explained in the table. Please refer to the following sections for more detail:
 
-### Match content id
+#### Match content id
```

Comment:
"id" in the subtitle plus in the text is not capitalised when used outside of a key. Above in the "DRM session in policy" section, it is capitalised.